src/node_options.cc

```diff
@@ -575,6 +575,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "write warnings to file instead of stderr",
             &EnvironmentOptions::redirect_warnings,
             kAllowedInEnvvar);
+  AddOption("[has_env_file_string]", "", &EnvironmentOptions::has_env_file_string);
+  AddOption("--env-file",
+            "load .env configuration file on startup (default: .env)",
```

Comment:
```suggestion
            "set environment variables from supplied file",
```

I don’t think it has a default value anymore, does it?

Reply:
Yes, this is old!